config/locales/candidate_interface/personal_statement.yml

```diff
@@ -1,6 +1,7 @@
 en:
   application_form:
     personal_statement:
+      label: Your personal statement
```

Comment:
Shouldn't we use it in the views as well then?

Reply:
I can always pick this up in my personal statement PR